scripts/deploy_ui.sh

```diff
@@ -9,26 +9,30 @@ export KUBECONFIG=${KUBECONFIG:-$HOME/.kube/config}
 export CONTAINER_COMMAND=${CONTAINER_COMMAND:-podman}
 export UI_DEPLOY_FILE=build/ui_deploy.yaml
 export UI_SERVICE_NAME=ocp-metal-ui
+export NO_UI=${NO_UI:-n}
```

Comment:
Why someone would want not to deploy the UI?

Reply:
cause u don't need it sometimes